### ISSUE TITLE
Four space indentation doesn't trigger indentation warning

### ISSUE THREAD

**Issue by dalen** [rodjek/puppet-lint#153]
This should trigger the warning "ERROR: two-space soft tabs not used on line 2", but it doesn't.

``` puppet
class foo {
    $bar = 'Hello world'
}
```

**Comment by rnelson0** [comment]
Verified this issue remains. It is not currently targeted against any release.

**Comment by rothsa** [comment]
Rolled some tests here: https://github.com/rothsa/puppet-lint/tree/2sp_soft_upgrade based on comments.
